bot/exts/evergreen/trivia_quiz/_game.py

```diff
@@ -0,0 +1,129 @@
+import asyncio
+import random
+
+import discord
+from discord.ext.commands import Context
+from fuzzywuzzy import fuzz
+
+from bot.bot import Bot
+from bot.exts.evergreen.trivia_quiz import _constants
+from bot.exts.evergreen.trivia_quiz import _helpers
+
+
+class Game:
+    """Class that contains information and functions about Tic Tac Toe game."""
+
+    def __init__(self, bot: Bot, ctx: Context, questions: dict, category: str):
+        self.bot = bot
+        self.ctx = ctx
+        self.questions = questions
+        self.category = category
+
+        # A variable to store the game status: either running or not running.
+        self.over = False
+        # A variable to store the person's ID who started the quiz game in a channel.
+        self.game_owner = ctx.author.id
+        # # A variable to store all player's scores for a the trivia quiz session.
+        self.player_scores = {}
+
+    async def play(self) -> None:
+        """Start and handle game."""
+        # Start game if not running.
+        if not self.over:
+            self.game_owner = self.ctx.author
```

Comment:
In `__init__`, `self.game_owner` is an `int` object, yet when `play` is called, `self.game_owner` is a `Member` object.